<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MyInvincibleLibrary</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imgresources.qrc">
    <normaloff>:/img/ic_launcher.png</normaloff>:/img/ic_launcher.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>800</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/img/background.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/img/myIDE_imagotype.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>1200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="addMetadata_button">
    <property name="geometry">
     <rect>
      <x>47</x>
      <y>105</y>
      <width>100</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:!hover{
	border-image: url(:/img/buttons/add_button.png);
}
QPushButton:hover{
	border-image: url(:/img/buttons/add_button_hover.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="consultMetada_button">
    <property name="geometry">
     <rect>
      <x>47</x>
      <y>180</y>
      <width>100</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:!hover{
	border-image: url(:/img/buttons/consult_button.png);
}
QPushButton:hover{
	border-image: url(:/img/buttons/consult_button_hover.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="removeMetadata_button">
    <property name="geometry">
     <rect>
      <x>47</x>
      <y>255</y>
      <width>100</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:!hover{
	border-image: url(:/img/buttons/remove_button.png);
}
QPushButton:hover{
	border-image: url(:/img/buttons/remove_button_hover.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="modifyMetadata_button">
    <property name="geometry">
     <rect>
      <x>47</x>
      <y>330</y>
      <width>100</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:!hover{
	border-image: url(:/img/buttons/modify_button.png);
}
QPushButton:hover{
	border-image: url(:/img/buttons/modify_button_hover.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>16</width>
      <height>721</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>420</y>
      <width>208</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>490</y>
      <width>208</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>8</y>
      <width>741</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="date">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>10</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Date</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>991</width>
      <height>671</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(235, 245, 238, 0.29);
border-radius: 10px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>536</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/img/server.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="storage_label">
    <property name="geometry">
     <rect>
      <x>73</x>
      <y>538</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Storage</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>550</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>991</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="filesView_button">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>85</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Info &amp; Grid View</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius: 20px;
}
QPushButton:!hover{
	background-color: rgb(85, 96, 128);
}
QPushButton:hover{
	background-color: rgb(66, 74, 96);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="imgresources.qrc">
      <normaloff>:/img/list.png</normaloff>:/img/list.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="filesSort_button">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>85</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius: 20px;
}
QPushButton:!hover{
	background-color: rgb(85, 96, 128);
}
QPushButton:hover{
	background-color: rgb(66, 74, 96);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="imgresources.qrc">
      <normaloff>:/img/upArrow.png</normaloff>:/img/upArrow.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="info_button">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>85</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius: 20px;
}
QPushButton:!hover{
	background-color: rgb(85, 96, 128);
}
QPushButton:hover{
	background-color: rgb(66, 74, 96);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="imgresources.qrc">
      <normaloff>:/img/info.png</normaloff>:/img/info.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="settings_button">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>85</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius: 20px;
}
QPushButton:!hover{
	background-color: rgb(85, 96, 128);
}
QPushButton:hover{
	background-color: rgb(66, 74, 96);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="imgresources.qrc">
      <normaloff>:/img/settings.png</normaloff>:/img/settings.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="menuButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>85</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:!hover{
	
	border-image: url(:/img/buttons/menu.png);
}
QPushButton:hover{
	border-image: url(:/img/buttons/menu_hover.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="loggerButton">
    <property name="geometry">
     <rect>
      <x>47</x>
      <y>433</y>
      <width>100</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:!hover{
	border-image: url(:/img/buttons/logger.png);
}
QPushButton:hover{
	border-image: url(:/img/buttons/logger_hover.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="imgresources.qrc"/>
 </resources>
 <connections/>
</ui>
